src/components/AccountDrawer/MiniPortfolio/constants.tsx

```diff
@@ -139,6 +139,35 @@ const TransactionTitleTable: { [key in TransactionType]: { [state in Transaction
   },
 }
 
+export const CancelledTransactionTitleTable: { [key in TransactionType]: string } = {
```

Comment:
Unfortunately the grammar is a bit different between i.e. "Swapped" vs "Swap" in the "Swap cancelled", so we cant just do "Swapped cancelled". It's probably also better for translations to have the full message translated as one line rather than two separate translations combined into one string